Protobuf-cc.podspec

```diff
@@ -0,0 +1,107 @@
+                    'src/google/protobuf/wire_format_unittest.cc',
+                    'src/google/protobuf/stubs/bytestream_unittest.cc',
+                    'src/google/protobuf/stubs/common_unittest.cc',
+                    'src/google/protobuf/stubs/int128_unittest.cc',
+                    'src/google/protobuf/stubs/io_win32_unittest.cc',
+                    'src/google/protobuf/stubs/once_unittest.cc',
+                    'src/google/protobuf/stubs/statusor_test.cc',
+                    'src/google/protobuf/stubs/status_test.cc',
+                    'src/google/protobuf/stubs/stringpiece_unittest.cc',
+                    'src/google/protobuf/stubs/stringprintf_unittest.cc',
+                    'src/google/protobuf/stubs/structurally_valid_unittest.cc',
+                    'src/google/protobuf/stubs/strutil_unittest.cc',
+                    'src/google/protobuf/stubs/template_util_unittest.cc',
+                    'src/google/protobuf/stubs/time_test.cc',
+                    'src/google/protobuf/stubs/type_traits_unittest.cc',
+                    'src/google/protobuf/io/coded_stream_unittest.cc',
+                    'src/google/protobuf/io/printer_unittest.cc',
+                    'src/google/protobuf/io/tokenizer_unittest.cc',
+                    'src/google/protobuf/io/zero_copy_stream_unittest.cc',
+                    'src/google/protobuf/util/type_resolver_util_test.cc',
+                    'src/google/protobuf/util/time_util_test.cc',
+                    'src/google/protobuf/util/json_util_test.cc',
+                    'src/google/protobuf/util/field_mask_util_test.cc',
+                    'src/google/protobuf/util/field_comparator_test.cc',
+                    'src/google/protobuf/util/delimited_message_util_test.cc',
+                    'src/google/protobuf/util/message_differencer_unittest.cc',
+                    'src/google/protobuf/util/internal/json_stream_parser_test.cc',
+                    'src/google/protobuf/util/internal/default_value_objectwriter_test.cc',
+                    'src/google/protobuf/util/internal/json_objectwriter_test.cc',
+                    'src/google/protobuf/util/internal/protostream_objectsource_test.cc',
+                    'src/google/protobuf/util/internal/protostream_objectwriter_test.cc',
+                    'src/google/protobuf/util/internal/protostream_objectwriter_test.cc'
+
+  s.header_mappings_dir = 'src'
+  s.libraries = 'c++'
+
+  s.ios.deployment_target = '7.0'
+  s.osx.deployment_target = '10.9'
+  s.tvos.deployment_target = '9.0'
+  s.watchos.deployment_target = '2.0'
+  s.requires_arc = false
+
+  s.pod_target_xcconfig = {
+    # Do not let src/google/protobuf/stubs/time.h override system API
+    'USE_HEADERMAP' => 'NO',
+    'ALWAYS_SEARCH_USER_PATHS' => 'NO',
+
+    # Configure tool is not being used for Xcode. When building, assume pthread is supported.
+    'GCC_PREPROCESSOR_DEFINITIONS' => '"$(inherited)" "COCOAPODS=1" "HAVE_PTHREAD=1"',
```

Comment:
What is `"COCOAPODS=1"` needed for?

Is `"HAVE_PTHREAD=1"` needed for should a `configure` have already taken care of things?

Reply:
`"COCOAPODS=1"` is not needed, I'll remove it. `"HAVE_PTHREAD=1"` is required because `configure` is not run when we install using cocoapods.